### ISSUE TITLE
Rename 'printchplenv' to 'printchplconfig'?

### ISSUE THREAD

**Issue by bradcray** [chapel-lang/chapel#18878]
In a discussion about the 'ChapelEnv' module today in which we were discussing how it wasn't
simply the environment that was being reasoned about (because some settings are obtained
through other means like default values, compiler flags, or `.config` files), it was suggested
that perhaps similarly `printchplenv` should be renamed to `printchplconfig` since we think
of the things printed out by this script as being about Chapel's configuration, not what's in the
user's environment per se.

**Comment by bradcray** [comment]
Potential counterarguments:
* maybe the Chapel environment is not literally a shell environment, but what Chapel infers its configuration to be, by whatever means?
* config could be confused with config vars, consts, types, params
* maybe this is not worth the effort

(where, in posting these, I'm not against the proposed renaming and do tend to like it better, but am also not sure that the current name is so bad to be worth the pain of switching)

**Comment by mppf** [comment]
> (where, in posting these, I'm not against the proposed renaming and do tend to like it better, but am also not sure that the current name is so bad to be worth the pain of switching)

I agree with that, but would lean towards switching if we rename ChapelEnv to ChplConfig, so that they could be consistent. I think this consistency makes it clearer what they are both about.

**Comment by ronawho** [comment]
I like the idea of renaming, but I'd also probably just leave `printchplenv` as a symlink in for a long time since we're so used to typing that.

**Comment by jhh67** [comment]
FWIW, I am often tricked by the "env" in `printchplenv` as I expect the variables to be in the environment. I would prefer `printchplconfig` (or `printchplcfg` or `prntchplcfg` depending on how much we hate vowels).
